server/app.py

```diff
@@ -94,14 +94,14 @@ def update_cache():
         new_stops.append(stop_info)
 
     for group in grouped_stops:
-        stop_info = get_stop_predictions(group.get('ids'), group.get('operator'), group.get('group_name'), group.get('use_destination_as_name', False))
+        for stop in group.get("stops", []):
+            stop_info = get_stop_predictions([stop.get('id')], stop.get('operator'), group.get('name'), group.get('use_destination_as_name', False))        
```

Comment:
i know i am nit picking here but lets make the args on their own lines, easier to read
```suggestion
            stop_info = get_stop_predictions(
                [stop.get('id')],
                stop.get('operator'),
                group.get('name'),
                group.get('use_destination_as_name', False),
            )
```